.github/workflows/build-ci.yaml

```diff
@@ -0,0 +1,77 @@
+name: build-ci-images
+
+on:
+  workflow_call:
+    secrets:
+      REGISTRY_LOGIN:
+        required: true
+      REGISTRY_PASSWORD:
+        required: true
+
+jobs:
+  warp10-ci:
+    uses: scality/workflows/.github/workflows/docker-build.yaml@1.0.2
```

Comment:
Just wondering, is there a reason to use specifically 1.0.2 here instead of v1?

Reply:
I need to add support for LFS, though it looks like the v1 tag has been retagged now so I'll change it over